packages/terra-search-field/src/SearchField.module.scss

```diff
@@ -95,6 +95,23 @@
     }
   }
 
+  .label {
+    color: var(--terra-search-field-label-color, #1c1f21);
+    display: block;
+    font-size: var(--terra-search-field-label-font-size, 1rem);
+    font-weight: var(--terra-search-field-label-font-weight, bold);
+    margin-bottom: var(--terra-search-field-label-margin-bottom, 0.357em);
+    text-align: left;
+  }
```

Comment:
Not a comment for your logic, but it is interesting that we have to customize the styles for a label instead of having a Terra Label component.